examples/platform/silabs/MatterConfig.cpp

```diff
@@ -234,7 +234,8 @@ CHIP_ERROR SilabsMatterConfig::InitMatter(const char * appName)
     ReturnErrorOnFailure(InitWiFi());
 
 #if CHIP_CONFIG_ENABLE_ICD_SERVER
-    ReturnErrorOnFailure(DeviceLayer::Silabs::WifiSleepManager::GetInstance().Init());
+    ReturnErrorOnFailure(DeviceLayer::Silabs::WifiSleepManager::GetInstance().Init(
+        &DeviceLayer::Silabs::WifiInterface::GetInstance(), &DeviceLayer::Silabs::WifiInterface::GetInstance()));
```

Comment:
suggestion. add using `chip::DeviceLayer::Silabs`
to make method call shorter
`WifiInterface::GetInstance()`

Reply:
done